src/components/button/selection/button-selection.test.tsx

```diff
@@ -0,0 +1,12 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+import {ButtonSelection} from './button-selection';
+
+describe(ButtonSelection.name, () => {
```

Comment:
I like this idea for different components for different variations of buttons! I'm not sure what the difference between this and the normal Button component are though, since `npm run storybook` isn't working locally on this branch at the moment.

Reply:
Uses `Toniq` prefix  for `ToniqButton` and added types for buttons